src/many-sources/abstract-pool.js

```diff
@@ -148,8 +154,8 @@ inherit(AbstractPool, Stream, {
     return index
   },
 
-  _removeCur(obs) {
-    if (this._active) {
+  _removeCur(obs, after) {
```

Comment:
I don't like passing a boolean, not sure about performance impact, but wouldn't it be better to have two methods?

`_removeCur` and `_removeCurWithDelayedUnsubscription`